TicketMate.Application/Requests/ProjectRequests/UpdateByGuid/UpdateProjectByGuidRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using TicketMate.Domain.Constants;
+using TicketMate.Domain.Validation.GuidValidation;
+using TicketMate.Domain.Validation.StringValidation;
+
+namespace TicketMate.Application.Requests.ProjectRequests.UpdateByGuid
+{
+    public class UpdateProjectByGuidRequest : IValidatable, IRequest
+    {
+        public UpdateProjectByGuidRequest() { }
+        public Guid Guid { get; set; }
+        public string? Name { get; set; }
+        public bool? IsActive { get; set; }
+
+
+        public bool IsValid(out Validator validator)
+        {
+            validator = new();
+
+            validator.ApplyRule(new GuidRequiredRule(Guid, nameof(Guid)));
+            if (Name != null)
+                validator.ApplyRule(new StringLengthLimitRule(Name, nameof(Name), MaxLength.ProjectName));

```

Comment:
Can we achieve the same result by passing isRequiredString = false into the constructor like below?
```
new StringLengthLimitRule(Name, nameof(Name), MaxLength.ProjectName, isRequiredString = false);
```

Your tests for the request object looked pretty good to me, so I think if you implement this change you can just re-run your tests to make sure it's applying the rule as expected.

Reply:
Resolving Comment as Mack pushed up this change - glad it worked out good @MackMcCall !